hypha/apply/templates/forms/includes/field.html

```diff
@@ -25,7 +25,10 @@
     {% endif %}
 
     {% if field.field.help_link %}
+        {% if field.field.only_input %}
```

Comment:
Seems like a leftover from old work as we do not use `only_input` now and separate `multi_input_field.html` is used.